config/config.go

```diff
@@ -236,3 +240,15 @@ func (c *Config) Validate() error {
 	// returns nil if no errors in slice
 	return errors.Join(e...)
 }
+
+var defaultHeadersToExtract = []string{
+	"User-Agent",
+}
+
+// getHTTPHeadersToExtract returns the list of HTTP headers to extract from a HTTP request/response
```

Comment:
This function is pretty straightforward so I'm not sure the comment is necessary as it's a bit redundant, though if we're adding a comment then maybe a bit more context would be useful

```suggestion
// getHTTPHeadersToExtract returns the list of HTTP headers to extract from a HTTP request/response
// based on a user-defined list in HTTP_HEADERS, or the default headers if no list is given. 
```